client/src/app/gateways/repositories/structure-levels/structure-level-repository.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+
+import { Identifiable } from '../../../domain/interfaces';
+import { StructureLevel } from '../../../domain/models/structure-levels/structure-level';
+import { ViewStructureLevel } from '../../../site/pages/meetings/pages/participants/pages/structure-levels/view-models/view-structure-level';
+import { BaseMeetingRelatedRepository } from '../base-meeting-related-repository';
+import { RepositoryMeetingServiceCollectorService } from '../repository-meeting-service-collector.service';
+import { StructureLevelAction } from './structure-level.action';
+
+@Injectable({
+    providedIn: `root`
+})
+export class StructureLevelRepositoryService extends BaseMeetingRelatedRepository<ViewStructureLevel, StructureLevel> {
+    public constructor(repositoryServiceCollector: RepositoryMeetingServiceCollectorService) {
+        super(repositoryServiceCollector, StructureLevel);
+    }
+
+    public getVerboseName = (plural?: boolean): string => (plural ? `StructureLevels` : `StructureLevel`);
```

Comment:
```suggestion
    public getVerboseName = (plural?: boolean): string => (plural ? _(`Structure levels`) : _(`Structure level`));
```

and `import {marker as _}` above

Reply:
done